runtime/doc/options.txt

```diff
@@ -3703,9 +3703,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 						*'list'* *'nolist'*
 'list'			boolean	(default off)
 			local to window
-	List mode: Show tabs as CTRL-I is displayed, display $ after end of
-	line.  Useful to see the difference between tabs and spaces and for
-	trailing blanks.  Further changed by the 'listchars' option.
+	List mode: Display non-printable characters according to 'listchars'.
```

Comment:
Is "non-printable" the right adjective here? Space is printable.

Reply:
We can use something else if it's clearer, but I don't think we should overthink it too much.

I'd honestly be fine with something as simple as "When set, display characters according to 'listchars'".